.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: ruby
 rvm:
-  - "2.0.0"
```

Comment:
We're not planning on dropping Ruby 2.0 at this time... but I'll discuss with the team.